resources/benchmark-runner.mjs

```diff
@@ -57,6 +57,10 @@ class Page {
         return null;
     }
 
+    forceReflowAndGetHeight() {
```

Comment:
I know reflow is the term of art for Firefox but WebKit/Blink uses Layout for the same concept.
How about being more explicit and calling it forceUpdateBoundingBoxes?

But this code may not be necessary after https://github.com/WebKit/Speedometer/pull/28 since all browsers implement HTML5 event loop's update the rendering step these days:
https://html.spec.whatwg.org/multipage/webappapis.html#update-the-rendering

Reply:
The goal is really to be able to use this in tests.

+1 for changing its name.

Reply:
I'd hate for browsers to start adding crazy optimizations around document.body.getBoundingRect though. It's not far fetched with the level of optimizations browsers are implementing these days. My preference is to defer to rAF to measure these things.

Reply:
I believe you're right.

Some colleagues mentioned another approach of relying on rAF (that would also have the nice benefit of allowing for Paint, which is what happens in real websites so the benchmark would be closer to these), but that would need the asynchronous runner.